tests/test_metadata_utils.py

```diff
@@ -251,6 +251,13 @@ def setUp(self) -> None:
                     },
                 ],
             },
+            {
+                "id": "0007",
+                "text": "Your article struck a cord, as a child I would imagine being an inventor. . As an adult I still love it when an insight to a new product or process reveals itself .",
+                "metadata": [
+                    {"key": "title", "type": "global", "value": "My Thoughts On It » Dad, I want to be an inventor"},
```

Comment:
Thank you very much for working to add this new processor. However, the titles are not directly formatted as you show in this example. 

Maybe, we can add also a `TitlePreprocessor` to process them to the format you show here?

If it helps, you can find examples on the dataset explorer [here](https://huggingface.co/spaces/bs-modeling-metadata/dataset-exploration).

Reply:
Hi @SaulLu, adding the `title` preprocessor in #134